saleor/graphql/giftcard/resolvers.py

```diff
@@ -1,13 +1,8 @@
-import graphene
-
-from ...core.tracing import traced_resolver
 from ...giftcard import models
-from .types import GiftCard
 
 
-@traced_resolver
-def resolve_gift_card(info, gift_card_global_id):
-    return graphene.Node.get_node_from_global_id(info, gift_card_global_id, GiftCard)
+def resolve_gift_card(gift_card_id):
```

Comment:
just a thought but do we have any convention for naming arguments of resolvers? In this PR i saw we sometimes use `id` and sometimes `thing_id`

Reply:
I think we don't have a naming convention but we can introduce it.